spec/services/plans/create_service_spec.rb

```diff
@@ -117,8 +118,19 @@
       it 'saves premium attributes' do
         plan = plans_service.create(**create_args).plan
 
-        expect(plan.charges.standard.first).to have_attributes(pay_in_advance: false, min_amount_cents: 100)
-        expect(plan.charges.graduated.first).to be_pay_in_advance
+        expect(plan.charges.standard.first).to have_attributes(
+          {
+            pay_in_advance: false,
+            min_amount_cents: 100,
+            invoiceable: true,
+          }
+        )
+        expect(plan.charges.graduated.first).to have_attributes(
+          {
+            pay_in_advance: true,
+            invoiceable: false,
+          }
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)